pkg/cli/create_platform.go

```diff
@@ -145,9 +145,9 @@ func CreatePlatform(ctx context.Context, options *CreateOptions, globalFlags *fl
 	}
 	log.Donef("Successfully created the virtual cluster %s in project %s", virtualClusterName, options.Project)
 
-	if options.CreateContext {
+	if options.Connect {
```

Comment:
I wonder if this block could just be removed

Reply:
I removed it. See the comment above.